data/deep jobs.txt

```diff
@@ -150,18 +150,19 @@ mission "Transport scientists"
 	repeat
 	to offer
 		random < 15
+	on offer
```

Comment:
This one I don't believe, for the sole reason of the scientists having government funding. Per diem limits exist now for reimbursable expenses, and I don't have any hope they'll disappear in the next 1000 years :rofl:

Reply:
Yeah I wavered on this one too, and eventually decided that the Wanderer scientists wouldn't mind normal digs. Deep folks aren't supposed to be real hoity-toity, so I can revert this.